chocolatey/docs
# (#997) Highlight Active page

## Description Of Changes
The change here ensures the left side navigation
always highlights the active page. This also
makes sure that an expanded navigation item does
not collapse on reload. The trailing slash, or
lack of, has been accounted for now.

## Motivation and Context
We want to ensure the left side navigation works.

## Testing


1. Pull down the pr
2. Run `yarn dev`, ensure the left side navigation is working properly. Try adding and removing slashes on the url to see that it just "works".
3. Close the preview terminal, and run `yarn build`
4. Run `yarn preview`
5. Try adding and removing slashes on the url to see that it just "works".

## Change Types Made


    * Issue #

## Change Checklist


## Related Issue
Relates to #997
